tests/logical_expressions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -13,10 +13,10 @@ const radioSelectedAttr =
 
 =====================================output=====================================
 const radioSelectedAttr =
-  (isAnyValueSelected &&
-    node.getAttribute(radioAttr.toLowerCase()) === radioValue) ||
-  (!isAnyValueSelected && values[a].default === true) ||
-  a === 0;
+  (isAnyValueSelected
+    && node.getAttribute(radioAttr.toLowerCase()) === radioValue)
+  || (!isAnyValueSelected && values[a].default === true)
+  || a === 0;
```

Comment:
Would it be possible to also change this case where the multiline operators are nested? When I encounter such a case in my code, I usually refactor it so that prettier doesn't make it ugly like that. If I was writing it manually I would write something like:

```JS
  (
    isAnyValueSelected
    && node.getAttribute(radioAttr.toLowerCase()) === radioValue
  )
  || (
    !isAnyValueSelected 
    && values[a].default === true
  )
  || a === 0;
```

Reply:
I'm trying to keep this change as small as possible to improve the odds the team decides to merge it. Based on the volume of :+1:s on #3806, putting binary operators at the beginning of lines has pretty widespread support. It helps that [the logic change](https://github.com/prettier/prettier/pull/7111/files#diff-3abca41914b4b214a2507e33cc54c28bL5725-R5729) to do that exclusively deletes code. I'm concerned that expanding the scope could reduce the chance it gets merged at all.

Reply:
Sure. I didn't mean this comment to show as a "review" of this PR. It was just a random question :sweat_smile: